content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,142 @@
+
+    {{< text bash >}}
+    $ export ORG_NAME=mymeshname.mycompanyname.com
+    {{< /text >}}
+
+1. Create a working directory where to store a number of files produced during the cluster
+onboarding process:
+
+    {{< text bash >}}
+    $ export WORKDIR=mydir
+    $ mkdir -p ${WORKDIR}
+    $ cd ${WORKDIR}
+    {{< /text >}}
+
+1. Download the setup script to your working directory:
+
+    {{< text bash >}}
+    $ curl {{< github_file >}}/samples/multicluster/setup-mesh >./setup-mesh.sh
+    $ chmod +x ./setup-mesh.sh
+    {{< /text >}}
+
+1. Create a new [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
+and create contexts for each cluster:
+
+    1. Create a new KUBECONFIG file:
+
+        {{< text bash >}}
+        $ touch kubeconfig.yaml
+        {{< /text >}}
+
+    1. Set the `KUBECONFIG` environment variable to point to the configuration file you downloaded:
+
+        {{< text bash >}}
+        $ export KUBECONFIG=./kubeconfig.yaml
+        {{< /text >}}
+
+    1. Add the clusters to the configuration file:
+
+        {{< text bash >}}
+        $ TBD
+        {{< /text >}}
+
+    1. Create contexts for all three clusters:
+
+        {{< text bash >}}
+        $ TBD
+        {{< /text >}}
+
+1. And finally, prepare the mesh. This will create a root key and certificate that will
+be used to secure communication between the clusters in the mesh:
+
+    {{< text bash >}}
+    $ ./setup-mesh.sh create-mesh
+    {{< /text >}}
+
+{{< warning >}}
+The final step above created some secret material in your working directory, in particular some private keys associated
+with the certificate. You should store and protect those secrets. TBD: what files precisely?
+{{< /warning >}}
+
+## Customizing Istio
+
+Creating the mesh above created a file called `base.yaml` in your working directory. This file defines the
+basic configuration that will be used when deploying Istio in your clusters (which will happen below). You
+can customize the `base.yaml` file if you'd like to change how Istio will be deployed in the cluster.
+
+TBD: What can be customized? Pointer to existing docs?
+
+## Adding and removing clusters
+
+Adding clusters to the mesh is easy:
+
+{{< text bash >}}
+$ ./setup-mesh.sh add-cluster ./kubeconfig.yaml context-east-0 network-east
+$ ./setup-mesh.sh add-cluster ./kubeconfig.yaml context-east-1 network-east
+$ ./setup-mesh.sh apply
+{{< /text >}}
+
+You provide the `setup-mesh` script with the name of the Kubernetes configuration file, the name of the context within the file,
+and the name of the network associated with the cluster in that context. This step will deploy Istio in the clusters and will
+wire the clusters together in a single mesh.
+
+You can easily remove a cluster from the mesh with:
+
+{{< text bash >}}
+$ ./setup-mesh.sh remove-cluster ./kubeconfig.yaml context-east-0
```

Comment:
Missing `./setup-mesh.sh apply` after the removal. There is the caveat that the removed cluster is not modified in the current script. The intermediate CA, control plane, etc. are left intact. The user still needs to clean these up.